test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -697,7 +708,7 @@ test
   });
 
 test
-  .meta('v3', false) // Not yet implemented in v3 (OKTA-548963)
+  .meta('v3', true)
```

Comment:
instead of changing to `true` you can delete this line instead. same with the others above